lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -30,7 +30,7 @@
                         breakpoints: this.breakpoints
                     });
 
-                    if ( this.model.get('name') ) {
+                    if ( this.model.get('programData').name ) {
```

Comment:
This test might be too fragile now. If programData is undefined in the model I think it will throw an error. Can you check whether this is the case?